src/timeline/InputBar.cpp

```diff
@@ -937,13 +940,34 @@ InputBar::command(const QString &command, QString args)
                                 cache::getMembers(this->room->roomId().toStdString(), 0, -1));
     } else if (command == QLatin1String("converttoroom")) {
         utils::removeDirectFromRoom(this->room->roomId());
+    } else if (command == QLatin1String("ignore")) {
+        this->setArgIgnored(args, true);
+    } else if (command == QLatin1String("unignore")) {
+        this->setArgIgnored(args, false);
     } else {
         return false;
     }
 
     return true;
 }
 
+void
+InputBar::setArgIgnored(const QString &user, const bool ignored)
```

Comment:
```suggestion
InputBar::toggleIgnore(const QString &user, const bool ignored)
```

It took me a while to understand, what you meant with this :D